tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenAssertDifferentLsidOnLastTwoCommands.cs

```diff
@@ -0,0 +1,57 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core;
+using MongoDB.Driver.Core.Events;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public class JsonDrivenAssertDifferentLsidOnLastTwoCommands : JsonDrivenTestRunnerTest
+    {
+        private readonly EventCapturer _eventCapturer;
+
+        public JsonDrivenAssertDifferentLsidOnLastTwoCommands(IJsonDrivenTestRunner testRunner, EventCapturer eventCapturer, Dictionary<string, object> objectMap)
+            : base(testRunner, objectMap)
+        {
+            _eventCapturer = eventCapturer;
+        }
+
+        public override void Act(CancellationToken cancellationToken)
+        {
+        }
+
+        public override void Assert()
+        {
+            var lastTwoCommands = _eventCapturer
```

Comment:
With my above suggestion, it should be just:

    filterFunc(_eventCapturer.Events)
    .Select(commandStartedEvent => ((CommandStartedEvent)commandStartedEvent).Command);

Reply:
I like how simple it is now.